pkg/connectors/connector_integration_test.go

```diff
@@ -22,10 +24,11 @@ func TestHealthz(t *testing.T) {
 }
 
 func TestCreateConnector(t *testing.T) {
-	client := NewClient(hostConnect)
+	client := connectors.NewClient(hostConnect)
+
 	resp, err := client.CreateConnector(
-		CreateConnectorRequest{
-			ConnectorRequest: ConnectorRequest{Name: "test-create-connector"},
+		connectors.CreateConnectorRequest{
+			ConnectorRequest: connectors.ConnectorRequest{Name: "test-create-connector" + uuid.New().String()},
```

Comment:
Out of curiosity: we use `uuid` because there would be conflict when replaying the tests multiple times?

Reply:
Indeed, before there was always a `docker-compose down` before running `make test-integration`
I chose to rework the test to avoid it by introducing a unique ID to guarantee test can be repeated on the same up and running instance.